R-packages/covidcast/R/covidcast.R

```diff
@@ -177,7 +177,9 @@ covidcast_signal <- function(data_source, signal,
                   .data$geo_type == given_geo_type)
 
   if (nrow(relevant_meta) == 0) {
-    relevant_meta <- list()
+    # even if no other metadata is available, we should set the geo_type so
+    # plotting functions can deal with this signal.
+    relevant_meta <- list(geo_type = geo_type)
```

Comment:
How much does this change the previous behavior?  Is it needed for testing?  All else equal, I would prefer that this PR only add tests.

Reply:
See the commit message in 85c1ca2. It addresses the last point in #239. This behavior has regressed several times, and regressed again in #156.

I could separate the fix into another PR, but basically I got halfway through writing the test when I realized the code had regressed again, so I decided to include the fix. Separating it out may be more trouble than it's worth for a one-liner.

Reply:
Sounds good to leave it in as is